### PR TITLE
fix: fix date you met update UX

### PR DESCRIPTION
**Pull request by simonardejr**
This is a quick fix for issue #[1927 ("Date you met" defaults to today upon edit, even if "I don't know" was selected previously)](https://github.com/monicahq/monica/issues/1927).

`$contact->first_met` doesn't exists, so, the condition is never met and the checkbox isn't checked. Also, `v-model` is always set to 'known' in `app.js`, making the last radio button checked by default.

### CODE REVIEW

File: resources/views/people/introductions/edit.blade.php [hunk 1]
```diff
@@ -67,9 +67,9 @@
                   {{-- You don't know the date you've met --}}
                   <div class="form-check">
                     <label class="form-check-label" for="is_first_met_date_unknown">
-                      <input type="radio" class="form-check-input" name="is_first_met_date_known" id="is_first_met_date_unknown" value="unknown"
-                      v-model="date_met_the_contact" value="unknown"
-                      {{ (is_null($contact->first_met)) ? 'checked' : '' }}
+                      <input type="radio" class="form-check-input" name="is_first_met_date_known" id="is_first_met_date_unknown" value="unknown" 
+                      @click="()=>{date_met_the_contact = 'unknown';}"
```
Comment (asbiin) [inline resources/views/people/introductions/edit.blade.php:71]: You may just write
```suggestion
                      @click="date_met_the_contact = 'unknown'"
```

File: resources/views/people/introductions/edit.blade.php [hunk 2]
```diff
@@ -81,8 +81,9 @@
                   {{-- You know the date you've met --}}
                   <div class="form-check">
                     <label class="form-check-label" for="is_first_met_date_known">
-                      <input type="radio" class="form-check-input" name="is_first_met_date_known" id="is_first_met_date_known" value="known" v-model="date_met_the_contact" value="known"
-                      {{ (! is_null($contact->first_met)) ? 'checked' : '' }}
+                      <input type="radio" class="form-check-input" name="is_first_met_date_known" id="is_first_met_date_known" value="known" 
+                      @click="()=>{date_met_the_contact = 'known';}"
```
Comment (asbiin) [inline resources/views/people/introductions/edit.blade.php:85]: You may just write
```suggestion
                      @click="date_met_the_contact = 'known'"
```

**Review by asbiin — CHANGES_REQUESTED** [review]
See my comments.
Please run `yarn prod` to build the assets too.

**Comment by simonardejr** [comment]
Don't know if I've done everything right (fetching, merge, and stuff), but let me know if I need to do anything else :grin:

**Comment by asbiin** [comment]
@simonardejr OK it *was* fine, but because of the merge of your other PR, this one needs to `git merge origin/master`, and run `yarn prod` again! (I know that's bothering...)

**Comment by simonardejr** [comment]
I've done  the merging and re-run `yarn prod`, but the build didn't change any files... Don't know if what I've done was right, if not, just let me know and I don't mind to start another clean PR. (sorry for my bad english)

**Comment by asbiin** [comment]
Thank you  @simonardejr

**Comment by simonardejr** [comment]
:grin: :+1:

**Pull request merged**